packages/hadron-react-bson/test/date-value.test.jsx

```diff
@@ -3,27 +3,6 @@ const { expect } = require('chai');
 const { shallow } = require('enzyme');
 const { DateValue } = require('../');
 
-describe('<DateValue />', () => {
```

Comment:
i think we still need these tests!

Reply:
Maybe I'm missing something but these tests are identical to the tests in the `without a timezone` describe block. I was thinking they were leftovers because they were added at different times. Happy to add them back in if I'm missing something